lib/shelley/bench/Restore.hs

```diff
@@ -258,6 +271,17 @@ cardanoRestoreBench tr c socketFile = do
                                   Benchmarks
 -------------------------------------------------------------------------------}
 
+data BenchResults = BenchResults
+    { qualifier :: Text
```

Comment:
I don't think it's obvious what `qualifier` is.

Can we clarify with at least say a
```
-- ^ e.g. "Seq 1% Wallet"
```
comment, or a more elaborate description?

Reply:
Also, since this module is a "restoration benchmark", it might be worth highlighting that yes indeed `BenchResults` contains more results than the restoration time "because it's interesting", or some better reason.